prepare/tasks/qa/multiple_choice/tasks.py

```diff
@@ -3,8 +3,14 @@
 
 add_to_catalog(
     FormTask(
-        inputs=["context", "context_type", "question", "choices"],
-        outputs=["answer", "choices"],
+        inputs={
+            "context": "str",
+            "context_type": "str",
+            "question": "str",
+            "choices": "List[str]",
+        },
+        outputs={"answer": "str", "choices": "List[str]"},
+        prediction_type="Any",
```

Comment:
I think the answer should be an "int", but I think right now the post_processor return a "str"